Source/Core/Core/NetPlayServer.cpp

```diff
@@ -40,6 +43,11 @@
 #include <arpa/inet.h>
 #endif
 
+#define HEAP_ALLOC(var, size)                                                                      \
+  lzo_align_t __LZO_MMODEL var[((size) + (sizeof(lzo_align_t) - 1)) / sizeof(lzo_align_t)]
+
+static HEAP_ALLOC(wrkmem, LZO1X_1_MEM_COMPRESS);
```

Comment:
Are you certain making this file-scoped is the best idea, rather than using it where applicable, or making it part of the server instance (i.e. using resources only when they're needed, rather than making it exist even if someone doesn't use netplay).

Reply:
It's some rather odd code which I copied from State.cpp. I can change to dynamically allocating the memory if you'd prefer.